apps/prairielearn/elements/pl-rich-text-editor/pl-rich-text-editor.py

```diff
@@ -100,6 +100,38 @@ def render(element_html: str, data: pl.QuestionData) -> str:
     )
     element_text = element_inner_html(element)
 
+    submitted_files = data["submitted_answers"].get("_files", [])
+    submitted_file = next(
+        (f for f in submitted_files if f.get("name", None) == file_name), None
+    )
+
+    if data["ai_grading"]:
+        if data["panel"] != "submission" or not submitted_file:
+            return ""
+
+        contents = submitted_file.get("contents", "")
+        contents = base64.b64decode(contents).decode("utf-8")
+
+        # MathJax gets embedded as `span.ql-formula` elements. This is fine;
+        # we'll just remove all attributes to save on tokens.
```

Comment:
Without this little bit, a MathJax expression ends up being rendered like this:

```html
<span data-value="1+1=2">$1+1=2$</span>
```

(The original has `class="ql-formula"`, but that's stripped out by [`stripHtmlForAiGrading(...)`](https://github.com/PrairieLearn/PrairieLearn/blob/d7446dfa11f6441c9e36f9023aec9292b83a44c5/apps/prairielearn/src/lib/ai-grading.ts#L10))

It's not too bad, and I'd be happy to remove this code if folks want to keep things dead simple.

Reply:
seems good to me